queries.sql

```diff
@@ -30,3 +30,96 @@ SELECT * FROM animals WHERE name != 'Gabumon';
 --Find all animals with a weight between 10.4kg and 17.3kg (inclusive):
 
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
+
+-- Begin the transaction
+BEGIN;
+
+-- Update the species column to "unspecified"
+UPDATE animals SET species = 'unspecified';
+
+-- Verify the change
+SELECT * FROM animals;
+
+-- Rollback the transaction
+ROLLBACK;
+
+-- Verify the species column reverted to its previous state
+SELECT * FROM animals;
+
+-- Begin the transaction
+BEGIN;
+
+-- Update the species column to "digimon" for animals with name ending in "mon"
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+
+-- Commit the transaction
+COMMIT;
+
+-- Verify the changes
+SELECT * FROM animals;
+
+-- Begin the transaction
+BEGIN;
+
+-- Update the species column to "pokemon" for animals without a species
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL OR species = '';
+
+-- Commit the transaction
+COMMIT;
+
+-- Verify the changes
+SELECT * FROM animals;
+
+-- Begin the transaction
+BEGIN;
+
+-- Delete all records from the animals table
+DELETE FROM animals;
+
```

Comment:
- [ ] Immediately after deleting all the records we must verify that the change was made using a `select` statement. kindly do so.
e.g
`SELECT COUNT(*) FROM ANIMALS;`

### Note that the screenshot for this transaction should bu updated as well to reflect the change